state/snapshotsManager.go

```diff
@@ -377,26 +377,23 @@ func (sm *snapshotsManager) snapshotUserAccountDataTrie(
 	}
 }
 
-type snapshotPruningStorer interface {
-	Put(key, val []byte) error
-	GetFromOldEpochsWithoutAddingToCache(key []byte) ([]byte, core.OptionalUint32, error)
-}
-
 func (sm *snapshotsManager) syncAccountCode(
 	codeHash []byte,
+	epoch uint32,
 	tsm common.StorageManager,
 ) error {
 	storer, ok := tsm.(snapshotPruningStorer)
 	if !ok {
 		return fmt.Errorf("invalid storer, type is %T", tsm)
 	}
 
+	// add separate func for trieStorageManager with this
```

Comment:
TODO

Reply:
And rename this func to `snapshotAccountCode` instead of `syncAccountCode`

Reply:
old comment, removed